packages/core/src/hooks/useWizard/__stories__/useWizard.stories.tsx

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import useWizard from "../useWizard";
+import { Button, Flex } from "../../../components";
+import Steps from "../../../components/Steps/Steps";
+import Heading from "../../../components/Heading/Heading";
+import { Decorator } from "@storybook/react";
+import styles from "./useWizard.stories.module.scss";
+
+const withUseWizardDecorator: Decorator = Story => (
+  <Flex direction={Flex.directions.COLUMN} gap={Flex.gaps.MEDIUM}>
+    <Story />
+  </Flex>
+);
+
+export default {
+  title: "Hooks/useWizard",
+  decorators: [withUseWizardDecorator]
+};
+
+export const Overview = {
+  render: () => {
+    const { activeStep, next, back, canGoNext, canGoBack } = useWizard({
+      stepCount: 5
+    });
+
+    return (
+      <>
+        <Heading weight={Heading.weights.MEDIUM} type={Heading.types.H3}>
+          Active Step: {activeStep}
+        </Heading>
+        <Flex gap={Flex.gaps.SMALL}>
+          <Button kind={Button.kinds.TERTIARY} onClick={back} disabled={!canGoBack}>
+            Back
+          </Button>
+          <Button onClick={next} disabled={!canGoNext}>
+            Next
+          </Button>
+        </Flex>
+      </>
+    );
+  }
+};
+
+export const WithInitialStep = {
+  render: () => {
+    const { activeStep, next, back, canGoNext, canGoBack } = useWizard({
+      initialStep: 2,
+      stepCount: 5
+    });
+
+    return (
+      <>
+        <Heading weight={Heading.weights.MEDIUM} type={Heading.types.H3}>
+          Active Step: {activeStep}
+        </Heading>
+        <Flex gap={Flex.gaps.SMALL}>
+          <Button kind={Button.kinds.TERTIARY} onClick={back} disabled={!canGoBack}>
+            Back
+          </Button>
+          <Button onClick={next} disabled={!canGoNext}>
+            Next
+          </Button>
+        </Flex>
+      </>
+    );
+  }
+};
+
+export const WithCompletion = {
```

Comment:
I think it's a bit redundant and can be in the overview story

Reply:
agree